### DISCUSSION TITLE
Can I manually update dtb / kernel by mounting partitions?

### DISCUSSION THREAD

**Discussion by arrow53** [OE4T/meta-tegra#1500] [General]
I'm on nanbield with an orin nano and external nvme. I don't think I can use capsules to update as my dtb / kernel are on an external drive.

I tried naively just trying to do this `mount /dev/nvme0n1p7 dtb` to put a new dtb file in place but mounting failed.

If I have a new dtb and kernel file I'm scp'ing over to my machine is there a manual way to put them in place?

Note, this is a follow up from a different [discussion](https://github.com/OE4T/tegra-demo-distro/discussions/292) but I marked that as answered as i thought I could use capsules

**Comment by arrow53** [comment]
Thinking about it I tried just dd'ing the files in place `dd if=tegra234-p3767-0003-p3768-0000-a0.dtb of=/dev/nvme0n1p7` and this works fine.

Is this an ok strategy for dtb / kernel?

**Reply by madisongh** [reply]
Should be OK.  Using a UEFI capsule for kernel/dtb updates is, IMO, a worse option.
